### PR TITLE
Scottish Microscopy Group Symposium 2017 workshop

### PR DESCRIPTION
**Pull request by will-moore**
For Wednesday 22nd Nov.

### CODE REVIEW

**Comment by jburel** [comment]
The concern with this PR is the changes to the share files e.g. css
that could affect other presentations
@will-moore reviewed some but not all

cc @sbesson

File: 2017/SMG-symposium-workshop/index.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+        <!-- For syntax highlighting -->
+        <link rel="stylesheet" href="reveal-js/lib/css/zenburn.css">
+
+        <!-- If the query includes 'print-pdf', use the PDF print sheet -->
+        <script>
+            document.write( '<link rel="stylesheet" href="reveal-js/css/print/' + ( window.location.search.match( /print-pdf/gi ) ? 'pdf' : 'paper' ) + '.css" type="text/css" media="print">' );
+        </script>
+
+        <!--[if lt IE 9]>
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: can this be removed?

File: 2017/SMG-symposium-workshop/index.html [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+                        <li>What is OMERO?</li>
+                        <li>Workshop: Using OMERO</li>
+                    </ul>
+                </section>
+
+
+                <section>
+                    <h2>A typical microscopy workflow</h2>
+
+                    <img src="http://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/typical-workflow.png">
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: https

File: 2017/SMG-symposium-workshop/index.html [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+                </section>
+
+                
+                <section>
+                    <h2>OME: Interoperability</h2>
+
+                    <img src="http://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/ome-informatics.png">
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: https

File: 2017/SMG-symposium-workshop/index.html [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+                    <h2>OME: What We Do</h2>
+
+                    <img src="images/ome-what-we-do.png">
+                </section>
+
+
+                <section>
+                    <h2>OMERO is a server with clients</h2>
+
+                    <!-- <img src="../../2014/technique_tapas_nov_5/images/omero-platform-simple.png"> -->
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: remove

File: 2017/SMG-symposium-workshop/index.html [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+                    <img src="http://downloads.openmicroscopy.org/presentations/2014/technique_tapas_nov_5/images/omero-platform-simple.png">
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: https

File: 2017/SMG-symposium-workshop/index.html [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+                </section>
+
+
+                <!-- Series of Insight screenshots -->
+                <section data-background="http://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/insightBg.png">
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: same

File: 2017/SMG-symposium-workshop/index.html [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+                </section>
+
+                <section data-background="http://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/insightBg.png">
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: same

File: 2017/SMG-symposium-workshop/index.html [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+                        <img src="http://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/insightImport.png"/>
+                </section>
+
+                <section data-background="http://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/insightBg.png">
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: same

File: 2017/SMG-symposium-workshop/index.html [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+                </section>
+
+
+                <section data-background="http://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/webclient3.png">
+                </section>
+
+<!--
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: remove

File: 2017/SMG-symposium-workshop/index.html [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+                <section>
+                    <h2>Demo</h2>
+                    <h3><a target="_blank" href="https://demo.openmicroscopy.org">demo.openmicroscopy.org</a></h3>
+                    <h3><code>Username: demo</code></h3>
+                    <h3><code contenteditable="true">Password: </code></h3>
+                </section>
+
+
+                <section>
+                    <h3><a target="_blank" href="http://idr.openmicroscopy.org/">idr.openmicroscopy.org</a></h3>
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: https

File: 2017/SMG-symposium-workshop/index.html [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+                    <iframe style="width:900px; height:600px" src="http://idr.openmicroscopy.org/about/"></iframe>
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: https

File: 2017/SMG-symposium-workshop/index.html [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+                </section>
+
+
+                <section>
+                    <h2>Thank you</h2>
+                    <div class="funding_logo_container">
+                        <img src="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/_images/logos/bbsrc.png" class="funding_logo" />
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: https

File: 2017/SMG-symposium-workshop/index.html [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+                        <img src="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/_images/logos/corbel.png" class="funding_logo" />
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: https

File: 2017/SMG-symposium-workshop/index.html [hunk 14]
```diff
@@ -0,0 +1,204 @@
...
+                        <img src="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/_images/logos/global_bioimaging.png" class="funding_logo" />
+                        <img src="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/_images/logos/horizon_2020.png" class="funding_logo" />
+                        <img src="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/_images/logos/multimot.png" class="funding_logo" />
+                        <img src="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/_images/logos/euro_bioimaging.png" class="funding_logo" />
+                    </div>
+                </section>
+
+
+                <section>
+                    <h3><a target="_blank" href="http://www.openmicroscopy.org/">openmicroscopy.org</a></h3>
```
Comment (jburel) [inline 2017/SMG-symposium-workshop/index.html]: https

**Comment by will-moore** [comment]
Actually it looks like the presentations in ```2017/Users-Meeting/Workshops/``` that share ```reveal``` are using the one in ```2017/Users-Meeting/Workshops/_reveal-js/``` not the base one in ```/template/``` updated in this PR. That is only used by ```2017/Team-Training/OMERO-web/``` which is looking fine.

**Comment by jburel** [comment]
Merging so we can add the url to the printout version

**Pull request merged**